### DISCUSSION TITLE
How do I make the thickness of plotted lines thicker using `set-style`?

### DISCUSSION THREAD

**Discussion by calebeby** [cetz-package/cetz-plot#45] [Q&A]
I would like the plotted lines (the actual data I'm graphing, not the axes or grid lines) to be thicker. I know that I can use `plot.add(..., style: (stroke: 4pt))` to make them thicker, but then I have to put that on every `plot.add` call. Is there a key in `set-style` corresponding to `plot.add`? I couldn't find documentation for it.

Thanks for your help! CeTZ is awesome!

**Comment by johannes-wolf** [comment]
I am moving this to the new home for cetz' plotting library, `cetz-plot`.

**Comment by johannes-wolf** [comment]
There is currently no default style for plot-data that you can set using `set-style`, but I think we should have this. I have created an issue: #46.
